src/splitInteger.test.js

```diff
@@ -2,7 +2,6 @@
 
 const splitInteger = require('./splitInteger');
 
-
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
   const numberToSplit = 12;
```

Comment:
You can simplify the test by directly calling splitInteger with parameters in the expect statement, avoiding the need for intermediate variable declarations (like numberToSplit, numberOfParts, and expectedParts)